src/ssl.c

```diff
@@ -16536,6 +16536,45 @@ int wolfSSL_i2d_PrivateKey(const WOLFSSL_EVP_PKEY* key, unsigned char** der)
     return wolfSSL_EVP_PKEY_get_der(key, der);
 }
 
+int wolfSSL_i2d_PrivateKey_bio(WOLFSSL_BIO* bio, WOLFSSL_EVP_PKEY* key)
```

Comment:
Should this function be in `src/bio.c`?

Reply:
I don't think so.  there are many functions in this file that look like `wolfSSL_i2d_*_bio()` and `wolfSSL_d2i_*_bio()` .